types/levelup/tslint.json

```diff
@@ -8,6 +8,10 @@
                     [
                         "NeedsExportEquals",
                         false
+                    ],
+                    [
+                        "NoDefaultExport",
```

Comment:
This should not be disabled unless there is a good reason for this. 
(not just for passing the CI tests)